README.md

```diff
@@ -1 +1,44 @@
-# lab-19
\ No newline at end of file
+# lab-19
+
+
+##  Message Queue
+
+### Author: Blandine Dasilveira
+
+### Links and Resources
+
+
+- queue-server:   will hold your queue server, running on port 3001. It should store some object/array containing the different queues in the system. When sockets connect to this socket server, a few actions should take place:
+
+- api server: This application will be both an Express API server running on port 3000 and a socket client connected to the queue server. It should have one primary route defined
+
+- vendor01 and vendor 02: You will have two applications representing two different vendors.
+
+
+
+- [submission PR](https://github.com/blandine-401javascript/lab-19/pull/1)
+
+
+### Setup
+
+#### `.env` requirements
+
+
+#### Tets
+
+
+
+
+
+- How to use your library?
+  -  open 4 console:
+    - run nodemon api.js
+    - run nodemon queue-sever.js
+    - run nodemon vendor-01.js
+    - run nodemon  vendor-02.js
+
+    use postman as a client to see the output in the console,
+    

```

Comment:
This part should be titled 'How to run application. I do like that you include using postman, but would add the path (example: delivery/flower-shop/2468)